src/external/wb_presets.c

```diff
@@ -2350,6 +2352,74 @@ const wb_data wb_preset[] =
   { "Nikon", "Coolpix P340", Cloudy, 0,			{ 1.976563, 1, 1.457031, 0 } },
   { "Nikon", "Coolpix P340", Flash, 0,			{ 2.085938, 1, 1.554688, 0 } },
 
+  // Nikon CoolPix P1000 firmware 1.2
+  { "Nikon", "COOLPIX P1000", HTMercury           , 0, { 1.76953125, 1, 1.61328125, 0 } },
+  { "Nikon", "COOLPIX P1000", DirectSunlight      , 0, { 1.4375, 1, 1.99609375, 0 } },
+  { "Nikon", "COOLPIX P1000", DirectSunlight      , 0, { 1.46875, 1, 1.94140625, 0 } },
+  { "Nikon", "COOLPIX P1000", DirectSunlight      , 0, { 1.50390625, 1, 1.88671875, 0 } },
+  { "Nikon", "COOLPIX P1000", DirectSunlight      , 0, { 1.5390625, 1, 1.8203125, 0 } },
+  { "Nikon", "COOLPIX P1000", DirectSunlight      , 0, { 1.578125, 1, 1.7578125, 0 } },
+  { "Nikon", "COOLPIX P1000", DirectSunlight      , 0, { 1.61328125, 1, 1.6953125, 0 } },
+  { "Nikon", "COOLPIX P1000", DirectSunlight      , 0, { 1.40625, 1, 2.05078125, 0 } },
```

Comment:
This causes finetuning for `DirectSunlight` to be not taken into consideration!
Also this is weird - it looks like you had -3..-2..-1..0..1..2..-4 based off of R&B coefficients.

Can you regenerate those?